### PR TITLE
Added url config

### PR DESCRIPTION
**Pull request by Shizuka0000**
Added url config option for networking, audio, acpi.
Custom profile can now also be configured using the url.
#448

### CODE REVIEW

File: src/browser/main.js [hunk 1]
```diff
@@ -842,10 +842,36 @@
             {
                 $("boot_options").style.display = "none";
 
+                parse_param()
                 start_emulation(settings, done);
             }
         }
 
+        function parse_param()
```
Comment (copy) [inline src/browser/main.js:850]: Could you pass settings to this function and move it to the outer scope (above `onload`)?

File: src/browser/main.js [hunk 2]
```diff
@@ -842,10 +842,36 @@
             {
                 $("boot_options").style.display = "none";
 
+                parse_param()
```
Comment (copy) [inline src/browser/main.js:845]: Missing a trailing semicolon.

File: src/browser/main.js [hunk 3]
```diff
@@ -842,10 +842,36 @@
...
+            const vram = parseInt(query_args["vram"], 10);
+            if(vram > 0)
+            {
+                settings.vga_memory_size = vram * 1024 * 1024;
+            }
+
+            const networking = parseInt(query_args["networking"], 10);
+            settings.networking = networking == 0 ? undefined : networking;
+
+            const audio = !!parseInt(query_args["audio"], 10);
```
Comment (copy) [inline src/browser/main.js:867]: I believe this should be called `disable_audio`. Alternatively, if you want to keep the name `audio` (which is a bit nicer), it should default to true.

File: src/browser/main.js [hunk 4]
```diff
@@ -842,10 +842,36 @@
...
+                settings.memory_size = Math.max(16, m) * 1024 * 1024;
+            }
+
+            const vram = parseInt(query_args["vram"], 10);
+            if(vram > 0)
+            {
+                settings.vga_memory_size = vram * 1024 * 1024;
+            }
+
+            const networking = parseInt(query_args["networking"], 10);
```
Comment (copy) [inline src/browser/main.js:864]: Should probably be called `networking_proxy` to make it a bit clearer.

**Comment by Shizuka0000** [comment]
Thanks for your reviews! I addressed the issues that you mentioned, please comment if there are any additional issues.

**Comment by copy** [comment]
Thanks!

**Pull request merged**